src/DotVVM.Framework/Resources/Scripts/tests/stateManagement.test.ts

```diff
@@ -141,7 +142,7 @@ test("Dirty flag", () => {
 })
 
 test("Upgrade null to observableArray", () => {
-    s.update(x => ({...x, ArrayWillBe: [ { $type: "t5", B: "ahoj" } ] }))
+    s.update(x => ({ ...x, ArrayWillBe: [{ $type: "t5", B: "ahoj" }] }))
```

Comment:
I know my rants on formatting just hide the important stuff, but these pointless formatting changes also hide the actual changes in the diff and clutter the history.

Reply:
Sorry about that, this wasn't intentional.